setup.cfg

```diff
@@ -1,2 +1,6 @@
 [metadata]
 description-file = README.md
+
+[build_manpages]
+manpages =
+    man/mtf.1:function=mtfparser:module=moduleframework.mtf_scheduler
```

Comment:
I guess, we have already several man pages. Do you plan to mention them?

Reply:
right, they can be generated too. I will create new issue about that.